main.tf

```diff
@@ -231,6 +231,20 @@ resource "google_compute_address" "static_nginx_ip" {
   }
 }
 
+resource "helm_release" "nginx_controller" {
```

Comment:
```suggestion
resource "helm_release" "nginx_controller" { # see https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
```

Reply:
Better place to put the comment will be in the next line.